src/main/java/com/iridium/iridiumskyblock/IridiumSkyblock.java

```diff
@@ -115,7 +126,10 @@ public void onEnable() {
         Bukkit.getServer().getOnlinePlayers().forEach(player -> getIslandManager().sendIslandBorder(player));
 
         addBstats(5825);
-        startUpdateChecker(62480);
+
+        if(getInstance().getConfiguration().updateChecks)
```

Comment:
I would move this check into the IridiumTeams method: https://github.com/Iridium-Development/IridiumTeams/blob/master/src/main/java/com/iridium/iridiumteams/IridiumTeams.java#L316